packages/react-components/react-aria/library/src/useTypingAnnounce/useTypingAnnounce.ts

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { useTimeout } from '@fluentui/react-utilities';
+import { useAnnounce, useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import type { AnnounceOptions } from '@fluentui/react-shared-contexts';
+import { AriaLiveAnnounceFn } from '../AriaLiveAnnouncer/AriaLiveAnnouncer.types';
+
+type Message = {
+  message: string;
+  options: AnnounceOptions;
+};
+
+const valueMutationOptions = {
+  attributes: true,
+  subtree: true,
+  characterData: true,
+  attributeFilter: ['value'],
+};
+
+export const useTypingAnnounce = (inputEl: React.RefObject<HTMLElement>): { typingAnnounce: AriaLiveAnnounceFn } => {
```

Comment:
Let's change API there to return a ref instead of accepting it i.e.

```
export const useTypingAnnounce = (): { typingAnnounce: AriaLiveAnnounceFn } => {
  const inputRef = React.useRef<HTMLElement>();
}
```

this way we will be able to refactor to callback refs without breaks later if needed